package.json

```diff
@@ -26,6 +26,7 @@
     "postcss-loader": "^1.3.3",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
+    "redux-entity-config": "^1.0.0",
```

Comment:
Maybe the confusion can be solved by improving the packages README? I'd be hesitant to have new devs do this manually as it could be a big time sink. If a concern is creating actions/reducers there is plenty of opportunity for that for everything else besides entities.